frontend/src/prompted_readings/prompted_readings.test.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './student_view';
+import SentimentScores from './analysis_view';
 
 it('renders without crashing', () => {
     const div = document.createElement('div');
     ReactDOM.render(<App />, div);
     ReactDOM.unmountComponentAtNode(div);
 });
+
+it('renders SentimentScores without crashing', () => {
```

Comment:
Fine for now -- going to merge, but better if you pass the component some data in this crash test (and ideally various edge case types of data)